### ISSUE TITLE
Cannot Scroll Suggested Results in Mobile Views

### ISSUE THREAD

**Issue by pmacMaps** [Esri/esri-leaflet-geocoder#198]
I discovered an odd behavior that appears to have been introduced at version 2.2.8 of the plugin.  The issue does not occur in version 2.2.6, and I discovered it using version 2.2.9.  

Whenever you are on a mobile device and type into the search control, and you try to scroll through the suggestions, it appears to fire on whatever suggestion the user presses. You cannot scroll to select a specific suggestion. 

I am using the Calcite Maps/Bootstrap projects, and encountered this issue on both an android and iPhone device.  The geosearch control is using the Esri World Geocoding Service and a Feature Layer search.

I captured some videos of the issues. Once they finish processing I'll add a comment with links.

**Apps**
Working app: http://maps.pamagic.org/membership-map/
Issues app: http://gis.ccpa.net/labs/pamagic/pamagic-members/
 
**Software/Plugins**
- Bootstrap 3.3.7
- jQuery 3.3.1
- Esri Calcite Maps 0.1
- Leaflet 1.3.1
- Esri Leaflet 2.1.4
- Esri Leaflet Geocoder 2.2.9 (2.2.8 issue as well)

**Comment by jgravois** [comment]
based on the [diff](https://github.com/Esri/esri-leaflet-geocoder/compare/v2.2.6...v2.2.8), my best guess is that the problem was introduced in https://github.com/Esri/esri-leaflet-geocoder/commit/d5de54b12f9b17e11c46db8d6cea767190420a2e

**Comment by pmacMaps** [comment]
Video of working app : https://youtu.be/SwrwWmnsksc

Video of broken app: https://youtu.be/Q5gsbZfbNos

**Comment by gavinr** [comment]
I did a bit of research on this. This issue looks to be due to the fact that we added the `touchend` event in d5de54b here: https://github.com/Esri/esri-leaflet-geocoder/commit/d5de54b12f9b17e11c46db8d6cea767190420a2e#diff-5763e4da84ffa56f56019ee1a0ff7aacR226 .... but the `touchend` event seems to actually be happening on `touchstart`, so that's why scrolling is not working (because we do the geosearch as soon as the finger touches the screen ... even if that finger is intending to do a scroll). Commenting out that `touchend` line seems to fix this issue (But I think would break #186 ?), so I'm not sure of a fix yet, but wanted to provide the info I had.

**Comment by jgravois** [comment]
i've got a proposed fix for this in #200 

for some reason `L.DomEvent.on(element, "touchend")` triggers a callback prematurely. 

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Esri Leaflet Geocoder</title>
    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1">

    <!-- Load Leaflet from their CDN -->
    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" />
    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet-src.js"></script>

    <!-- Make the map fill the entire page -->
    <style>
      #map {
        position: fixed;
        top: 0;
        bottom: 0;
        left: 0;
        right: 0;
      }
    </style>
  </head>
  <body>
    <div id="map"></div>
    <script>
      const map = document.getElementById('map');

      map.addEventListener('touchend', function (e) {
        console.log('native event', e);
      });

      L.DomEvent.on(map, "touchend", function(e) {
        console.log('leaflet event', e);
      })
    </script>
  </body>
</html>
```

![touchend](https://user-images.githubusercontent.com/3011734/41385229-0ffa0536-6f2f-11e8-910d-72165a990bb3.gif)

likely related: https://github.com/Leaflet/Leaflet/issues/5745

**Comment by jgravois** [comment]
released as [`v2.2.10`](https://github.com/Esri/esri-leaflet-geocoder/releases/tag/v2.2.10) :tada:
